zamba/object_detection/yolox/megadetector_lite_yolox.py

```diff
@@ -132,18 +134,20 @@ def _preprocess_video(self, video: np.ndarray) -> np.ndarray:
         """Process a video for the model, including resizing the frames in the video, transposing
         from (batch, height, width, channel) to (batch, channel, height, width) and casting to float.
         """
-        resized_frames = []
+        resized_video = np.zeros(
+            (video.shape[0], video.shape[3], self.config.image_height, self.config.image_width),
+            dtype=np.float32,
```

Comment:
Are these floats at this point? May be worth double checking since a lot of times image data gets loaded in as `unit8`.

Reply:
the image is loaded as a float here: https://github.com/drivendataorg/zamba/blob/master/zamba/object_detection/yolox/megadetector_lite_yolox.py#L107

but the output of `_preprocess` is a float: https://github.com/drivendataorg/zamba/blob/master/zamba/object_detection/yolox/megadetector_lite_yolox.py#L115-L124

AFAIC this is the correct input for mdlite